builq.go

```diff
@@ -68,7 +82,8 @@ func (a *argument) Format(s fmt.State, v rune) {
 		a.forQuery = true
 		fmt.Fprint(s, a.pp.Next())
 	default:
-		panic(fmt.Sprintf("unsupported verb %c", v))
+		a.err = fmt.Errorf("builq: unsupported verb %c", v)
+		// panic(a.err) // this panic will be caught and written to s by the fmt code.
```

Comment:
Now I'm not sure we need panic because a user won't see a query if the error is occurred anyway. @cristaloleg WDYT?